devservices/config.yml

```diff
@@ -9,9 +9,20 @@ x-sentry-service-config:
         repo_name: sentry-shared-kafka
         branch: main
         repo_link: https://github.com/getsentry/sentry-shared-kafka
+    taskbroker:
+      description: Taskbroker service
   modes:
     default: [kafka]
-    containerized: [kafka]
+    containerized: [kafka, taskbroker]
+
+services:
+  taskbroker:
+    image: ghcr.io/getsentry/taskbroker:latest
+    environment:
+      TASKBROKER_KAFKA_CLUSTER: "kafka-kafka-1:9093"
```

Comment:
How does this address get set correctly? I notice it's different from the one defined in snuba, and different from the name of the current kafka devservice (which is called `sentry_kafka`).

Reply:
Yes, taskbroker will error and crash if it cannot connect to Kafka.